src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerGetAdminMediasIntTest.java

```diff
@@ -149,13 +149,13 @@ void transformedMediaNotProvided() throws Exception {
 
         // when
         MvcResult mvcResult = mockMvc.perform(get(ENDPOINT_URL)
-                                                  .queryParam("transformed_media_id", ""))
+            )
             .andExpect(status().isBadRequest())
             .andReturn();
 
         // then
         String actualJson = mvcResult.getResponse().getContentAsString();
-        assertTrue(StringUtils.contains(actualJson, "Either transformed_media_id or transcription_document_id must be provided in the request, but not both."));
+        assertTrue(StringUtils.contains(actualJson, "Required request parameter 'transformed_media_id' for method parameter type Integer is not present"));
```

Comment:
Maybe nice to unmarshall and check on the correct error code also e.g.

Problem problemResponse = objectMapper.readValue(content, Problem.class);
        assertEquals(TranscriptionApiError.TRANSCRIPTION_DOCUMENT_HIDE_ACTION_REASON_NOT_FOUND.getType(), problemResponse.getType());

Reply:
This error is auto generated from Spring and the openapi "required" parameter.
the whole error looks like this:-
```json
{
  "title": "Bad Request",
  "status": 400,
  "detail": "Required request parameter 'transformed_media_id' for method parameter type Integer is not present"
}
```

And so doesn't have a type.